app/src/main/java/com/github/wnder/picture/ExistingPicture.java

```diff
@@ -121,4 +122,23 @@ public void onSuccess(DocumentSnapshot documentSnapshot) {
         });
     }
 
+    /**
+     * Returns the picture's location with purposely reduced precision
+     * @return the location
+     */
+    public void onApproximateLocationAvailable(Consumer<Location> approximateLocationAvailable) {
+        onLocationAvailable((location) -> {
+            double radius = 200; // meters
```

Comment:
Wouldn't it be better to pass this as argument because at some point we would like to tune the difficulty ?